src/SW_netCDF.c

```diff
@@ -1297,12 +1334,21 @@ static void create_netCDF_var(
         return; // Exit prematurely due to error
     }
 
+    if (!isnull(chunkSizes)) {
+        if (nc_def_var_chunking(*ncFileID, *varID, NC_CHUNKED, chunkSizes) !=
+            NC_NOERR) {
+            LogError(LogInfo, LOGERROR, "Just here.", varName);
```

Comment:
Could you please update the message on failed chunking to include a bit more information, e.g., "Failed to chunk variable '%s' in output netCDF."